test/auth/auth.test.js

```diff
@@ -0,0 +1,63 @@
+const chai = require('chai'),
+  { expect } = chai;
+const chaiHttp = require('chai-http');
+const userModelManager = require('../../lib/modelManagers/usermodel');
+const authController = require('../../controllers/AuthController');
+const server = require('../../index');
+
+chai.use(chaiHttp);
+chai.should();
+
+let newUser;
+
+const next = () => {};
+const res = () => {};
+
+/*
+ Test to make sure User is created and token is generated
+*/
+
+describe('Test for adding a new user and add token to request', () => {
+  describe('Test with correct information', () => {
+    const req = {
+      body: {
+        email: 'testmail@gmail.com',
+        password: '111111',
+        firstname: 'testfirstname',
+        lastname: 'testlastname'
+      }
+    };
+    const correctDetails = {
+      email: 'testmail@gmail.com',
+      password: '111111'
+    };
+
+    it('Should add a new user to the database', async () => {
+      await authController.signUp(req, res, next);
```

Comment:
what are we asserting for here?
` await authController.signUp(req, res, next);`

Reply:
I'm testing if a new user is added to the database that's why i'm immediately checking if the added user exist after running that function

Reply:
@tosinibrahim96 you can move `await authController.signUp(req, res, next); to a 'before' function instead performing the operation in 'it'.

Reply:
i have done that @eltNEG